docs/getting_started/install.rst

```diff
@@ -71,7 +71,7 @@ Installing the Development Version from Github
 You can install the latest development version (not as well tested) from the
 Github main branch:
 
-    | >> conda create -n <env_name> python=3.11
```

Comment:
I think we need to keep the `python` specification, just remove the version pin. Sorry for the imprecise request.

Reply:
To clarify, do you want this to say `python=3.13` or `python=3.12`? I don't think an unversioned qualifer (`python`) is supported (but I'm not a conda user so am not sure).

Reply:
In the past, we did have an unqualified `conda create -n <env_name> python`, but I'm fine avoiding that to prevent undefined behavior. We can use either, but let's do 3.12 just because it's been thoroughly tested? 🤷

Reply:
We've gathered some conflicts during my lack of a response - once this is clean I'll get it merged.